### PR TITLE
Development

### PR DESCRIPTION
**Pull request by danyonedwards**
My dev branch is working correctly and is ready to merge.

### CODE REVIEW

**Review by codevski — APPROVED** [review]
Top Bloke

**Review by blakehowe — COMMENTED** [review]
Really good job, keep up the good work (y)

**Review by johnnylak — APPROVED** [review]
Could do better...

File: helloworld/.idea/compiler.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ladybunne) [inline helloworld/.idea/compiler.xml:1]: @Auphonix `.idea` files shouldn't be included in the repo. Add an entry in your `.gitignore` to remove these, please!

**Review by ladybunne — CHANGES_REQUESTED** [review]
Add a couple of things to your `.gitignore` so we don't fill the repo with trash. `.DS_Store` and `.idea` in particular should never end up in a repo.

**Comment by blakehowe** [comment]
Really good job, keep up the good work (y)

**Pull request closed without merging**